README.md

```diff
@@ -40,6 +40,16 @@ Add Orbit package to your project by adding the package dependency:
 
 ## Usage
 
+0. (Optional) Set custom fonts. [Circular Pro is only licensed for Kiwi.com domain, for non-Kiwi.com projects use an alternative](https://orbit.kiwi/foundation/typography/open-source/).
+
+```swift
+Font.orbitFonts = [
+    .regular: URL(fileURLWithPath: ...),
+    .medium: URL(fileURLWithPath: ...),
+    .bold: URL(fileURLWithPath: ...)
```

Comment:
We could give an exact hint (to see it is a full path), such as URL(fileURLWithPath: "file://User/Joe/Developer/orbit-swift-ui/Foundation/Typography/CircularPro-Bold.otf").